test/EstateRegistry.js

```diff
@@ -338,6 +342,22 @@ contract('EstateRegistry', accounts => {
       await assertEstateSize(estateId, 2)
     })
 
+    it('transfering tokens in fires the AddLand event', async function() {
```

Comment:
Super minor sorry, but `emits` instead of `fires` could be more appropriate due we `emit Event()`